.github/workflows/build-pr.yml

```diff
@@ -55,6 +55,7 @@ jobs:
     strategy:
       matrix:
         KUBERNETES_VERSION: ["1.24.6"]
+        GATEKEEPER_VERSION: ["3.10.0"]
```

Comment:
There some discussion around 3.11 being released , and external provider is now in beta. Just curious, Will be continue testing 3.10 if we move to 3.11?

Reply:
wondering are they compatible

Reply:
GK supports n-1 versions, so ideally it should be tested with 2 versions.

Reply:
> There some discussion around 3.11 being released , and external provider is now in beta. Just curious, Will be continue testing 3.10 if we move to 3.11?

good call, I'll support both though they're not compatible.